<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>comptes</class>
 <widget class="QMainWindow" name="comptes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="retrait">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#retrait{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Retrait</string>
    </property>
   </widget>
   <widget class="QPushButton" name="versement">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#versement{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Versement</string>
    </property>
   </widget>
   <widget class="QPushButton" name="download">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#download{
	background-color:#eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>hISTOGRAMME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_title{
	color: green;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string>GESTION DES COMPTES</string>
    </property>
   </widget>
   <widget class="QLabel" name="searchTitle">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de compte:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultat de votre recherche</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>180</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>240</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>120</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchClients">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#search{
		background-color: #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>360</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="transaction">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#transaction{
           font-size: 14px;
		   color: green;
          background: rgb(170, 170, 127);
}</string>
    </property>
    <property name="text">
     <string>                             TRANSACTION</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>188</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Numero:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="compteAffiche"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>188</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameAfiiche"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>188</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Solde:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="soldeAffiche"/>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>661</width>
      <height>301</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="transferer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#transferer{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>Transfert entre deux Comptes</string>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="title_etat">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#title_etat{
	color: green;
	font-size: 16px;
}</string>
    </property>
    <property name="text">
     <string>ETAT DES MOUVEMENTS BANCAIRES D' UN CLIENT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>date(yyyy-mm-dd):</string>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#back{
	background-color:  #eeeeee;
    color: green;
}</string>
    </property>
    <property name="text">
     <string>&lt;&lt;Precedent</string>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>540</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="debit">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>550</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#debit{
	color: green;
	font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>DEBIT =</string>
    </property>
   </widget>
   <widget class="QLabel" name="debitTotal">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>550</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#debitTotal{
	color: red;
	font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="credit">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>550</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#credit{
	color: green;
	font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>CREDIT =</string>
    </property>
   </widget>
   <widget class="QLabel" name="creditTotal">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>550</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#creditTotal{
	color: red;
	font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>470</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="nombreRetrait">
       <property name="styleSheet">
        <string notr="true">#nombreRetrait{
	color: green;
	font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Total retrait:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numberRetrait">
       <property name="styleSheet">
        <string notr="true">#numberRetrait{
	color: red;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>110</width>
      <height>16</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="nombreVersement">
       <property name="styleSheet">
        <string notr="true">#nombreVersement{
	color: green;
	font-size: 12px;
}</string>
       </property>
       <property name="text">
        <string>Total Versement:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numberVersement">
       <property name="styleSheet">
        <string notr="true">#numberVersement{
	color:red;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="datei">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="datef">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>590</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="histogramme">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="alerte">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>590</y>
      <width>651</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#alerte{
	color: green;
	font-size: 17px;
}</string>
    </property>
    <property name="text">
     <string>l</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
